test/quantization/core/test_quantized_op.py

```diff
@@ -658,7 +656,6 @@ def test_qthreshold(self, X, threshold, value):
         ops_under_test = {
             'native': torch.threshold,
             'nn.functional': torch.nn.functional.threshold,
-            'nn.quantized.functional': torch.ao.nn.quantized.functional.threshold,
```

Comment:
Both these names are valid for this function. So I don't think it is ok to remove this @kit1980

Reply:
@albanD But the key of this dict - name  - is used only for printing, it doesn't test anything in addition to `'ao.nn.quantized.functional': torch.ao.nn.quantized.functional.threshold`:

        for name, op in ops_under_test.items():
            qY = op(qX, threshold, value)
            self.assertEqual(qY, qY_hat, msg=f"{name} qthreshold failed")